css-layout-classes/styles.css

```diff
@@ -0,0 +1,22 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+}
+
+.column {
+  width: 100%;
+}
```

Comment:
It seems you are reusing this `column` class for a lot of different elements with differing sizes. You should make specific `column` classes for each size necessary for the output. You will likely need a `column-third` and `column-one-fourth`